src/tech/v3/libs/smile/nlp.clj

```diff
@@ -101,3 +108,52 @@
        1000
        #(bow->sparse-fn % vocab->index-map)
        (get ds bow-col))))))
+
```

Comment:
The following functions implement the "most frequent used" tfidf calculation.
There are more alternatives, maybe to be added in future:
https://en.wikipedia.org/wiki/Tf%E2%80%93idf

I implemented exatly the "Example of tf–idf" of teh wikipedia article.

Reply:
This is really great work overall.